charts/athens-proxy/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: athens-proxy
-version: 0.2.0
+version: 0.2.1
 appVersion: 0.2.0
```

Comment:
@rimusz I bumped `version` to `0.2.1`. But for `appVersion`, shouldn't it be the version of athens image: https://github.com/gomods/athens/blob/master/charts/athens-proxy/values.yaml#L5?

Reply:
yes, app version has to be the same as the image tag

Reply:
@rimusz Done